common/compiler/solidity.go

```diff
@@ -54,6 +49,15 @@ var (
 		"--add-std",
 		"1",
 	}
+	paramsNew = []string{
+		"--bin",        // Request to output the contract in binary (hexadecimal).
+		"--abi",        // Request to output the contract's JSON ABI interface.
+		"--userdoc",    // Request to output the contract's Natspec user documentation.
+		"--devdoc",     // Request to output the contract's Natspec developer documentation.
+		"--add-std",    // include standard lib contracts
+		"--optimize=1", // code optimizer switched on
```

Comment:
Solidity can accept two different "optimize" parameters, both of which accept an integer value, hence the desire to pass it through from Geth.

```
--optimize arg (=0)                   Optimize bytecode
--optimize-runs arg (=200)            Estimated number of contract runs for optimizer.
```